### DISCUSSION TITLE
Saturation/Response Curves as a dataframe

### DISCUSSION THREAD

**Discussion by xijianlim** [facebookexperimental/Robyn#343] [General]
I would like a way to obtain the saturation/response curves as a dataframe. Currently, the only method presented in the demo is a one-for-one response using this method which returns list consisting of a a single value and a plot.

`imps <- 5000000
response_imps <- robyn_response(
  robyn_object = robyn_object
  #, select_build = 1
  , media_metric = "display_impressions"
  , metric_value = imps
    )`
I would like to get the plot as a dataframe instead or get ALL media variables saturation plots as a dataframe.

**Comment by laresbernardo** [comment]
Hi @xijianlim You could actually hack the `robyn_response()` outputs and fetch the data points you need from the `ggplot` object: `response_imps$plot$data`
